koios_python/asset.py

```diff
@@ -156,7 +160,10 @@ def get_policy_asset_list(self, asset_policy):
     :rtype: list.
     """
     timeout = get_timeout()
-    info = requests.get(f"{self.POLICY_ASSET_LIST_URL}{asset_policy}", timeout=timeout)
+    custom_headers = {"Range": str(content_range)}
```

Comment:
I think this function does not use the `content range`, at least that is what I thought from the Koios API endpoint here https://api.koios.rest/#get-/policy_asset_list